src/supporting-data/taxonomy/adapters/taxonomyConverter.ts

```diff
@@ -16,19 +19,29 @@ type Strain = {
   name: string;
 };
 
-type TaxonomyLight = {
-  taxonId: number;
-  scientificName: string;
-  synonyms?: string[];
-  commonName?: string;
-};
+export type Lineage = Array<SetRequired<TaxonomyDatum, 'hidden' | 'rank'>>;
```

Comment:
Maybe there are even more fields that can be removed in this case?
```suggestion
export type Lineage = Array<SetRequired<Omit<TaxonomyDatum, 'lineage'>, 'hidden' | 'rank'>>;
```

Thinking there might be more places around this file where the use of `Omit` might be good, so that we don't conditionally do stuff on fields that we should know are going to be `never` (hosts? strains? Anything that is a call to a recursive type of TaxonomyDatum could be candidate)

Reply:
Not sure, I started playing a bit with it but think it might be more pain than what it gives us, especially in the long run if we have to do this everywhere.

Reply:
Well I'd do at least the one in the suggestion then, because this is an obvious one.